stagemonitor-core/src/main/java/org/stagemonitor/core/elasticsearch/ElasticsearchClient.java

```diff
@@ -73,6 +74,17 @@ public ElasticsearchClient(final CorePlugin corePlugin, final HttpClient httpCli
 		}
 	}
 
+	public boolean isElasticsearch6() {
```

Comment:
```java
isElasticsearch6Compatible() {
    return esMajorVersion >= 6;
}
```

Elasticsearch 7 will most likely be compatible with ES6. But with the current logic, `getElasticsearchResourcePath` would return `"kibana/5/"` for ES7 while `"kibana/7/"` would be more appropriate.